packages/react/src/Accordion/Accordion.tsx

```diff
@@ -15,15 +15,19 @@ export type AccordionProps = {
   /**
    * The hierarchical level of this Accordion’s Section Headings within the document.
    * There is no default value; determine the correct level for each instance.
-   * The size of the heading is fixed at level 3.
    */
-  headingLevel: HeadingProps['level']
+  headingLevel: Exclude<HeadingProps['level'], 1>
+  /** Uses larger or smaller text without changing its position in the heading hierarchy. */
+  headingSize?: Exclude<HeadingProps['size'], 'level-1' | 'level-6'>
```

Comment:
We could consider deprecating these values instead of changing them – we don’t have any other breaking changes yet. Let’s discuss on Wednesday.